### PR TITLE
Feature/mammals objects

### PR DESCRIPTION
**Pull request by AwdamnKaya**
created mammals objects and integrated into sidebar

### CODE REVIEW

File: scripts/mammals.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+                </a>
+                <span class="tooltip">${mammal.name}</span>
+            </li>
+    `
+});
+
+let animalButtons = Array.from(document.querySelectorAll(".animal"));
+
+animalButtons.forEach(b => b.addEventListener('click', ()=> {
+    let activeAnimal = animalButtons.find(b => b.classList.contains("animal-active"));
```
Comment (Alexander-Lindstroem) [inline scripts/mammals.js]: I would recommend using a more syntactic variable name, but it won't change the outcome of the code so if you don't want to you don't have to.

**Review by Alibaba000101 — APPROVED** [review]
Cool

**Review by Alexander-Lindstroem — CHANGES_REQUESTED** [review]

**Review by andre84o — APPROVED** [review]
LG

**Review by Alexander-Lindstroem — APPROVED** [review]
Good changes.

**Pull request merged**